### PR TITLE
Add reverse-bridging benchmarks for array subscripting

### PR DESCRIPTION
**Pull request by Catfish-Man**

### CODE REVIEW

**Comment by Catfish-Man** [comment]
@swift-ci please smoke benchmark

**Comment by Catfish-Man** [comment]
@swift-ci please smoke test

File: benchmark/single-source/ObjectiveCBridging.swift [hunk 1]
```diff
@@ -79,6 +79,18 @@ public let ObjectiveCBridging = [
   BenchmarkInfo(name: "UnicodeStringFromCodable",
                  runFunction: run_UnicodeStringFromCodable, tags: ts,
                  setUpFunction: setup_UnicodeStringFromCodable),
+  BenchmarkInfo(name: "BridgedNSArrayObjectAtIndex",
```
Comment (lorentey) [inline benchmark/single-source/ObjectiveCBridging.swift]: These names would read a little nicer with components separated by dots, as in `BridgedNSArray.objectAtIndex`, `BridgedNSArray.mutableCopyObjectAtIndex` etc.

File: benchmark/single-source/ObjectiveCBridging.swift [hunk 2]
```diff
@@ -773,3 +785,73 @@ public func run_UnicodeStringFromCodable(_ N: Int) {
...
+  nsArray = NSArray(objects: &arr, count: 100)
+  nsArrayMutableCopy = (nsArray.mutableCopy() as! NSMutableArray)
+  #endif
+}
+
+@inline(never)
+public func run_BridgedNSArrayObjectAtIndex(_ N: Int) {
+  #if _runtime(_ObjC)
+  for _ in 0 ..< N {
+    for _ in 0..<50 {
```
Comment (lorentey) [inline benchmark/single-source/ObjectiveCBridging.swift]: Is there a reason for the nested loops here? We usually just go from `0` to `50 * N` elsewhere. Overflows are only a concern on 32-bit platforms, and the benchmark runner can work around them by limiting the iteration count.

**Review by lorentey — APPROVED** [review]
✨ A++++ would review again ✨

File: benchmark/single-source/ObjectiveCBridging.swift (hunk 2 above)
Comment (Catfish-Man) [reply benchmark/single-source/ObjectiveCBridging.swift]: oh, right. Just wasn't thinking :)

File: benchmark/single-source/ObjectiveCBridging.swift (hunk 1 above)
Comment (Catfish-Man) [reply benchmark/single-source/ObjectiveCBridging.swift]: Sure, sounds good

**Comment by Catfish-Man** [comment]
@swift-ci please smoke test and merge

**Comment by Catfish-Man** [comment]
@swift-ci please smoke test and merge

**Comment by Catfish-Man** [comment]
@swift-ci please test linux platform

**Comment by Catfish-Man** [comment]
@swift-ci please smoke test linux platform

**Pull request merged**